lib/membrane_rtmp_plugin/rtmp/source/source.ex

```diff
@@ -143,9 +190,16 @@ defmodule Membrane.RTMP.Source do
   end
 
   def handle_info(:start_receiving, _ctx, %{socket_retries: retries} = state) do
-    case :gen_tcp.controlling_process(state.socket, state.receiver_pid) do
+    module = if state.use_ssl?, do: :ssl, else: :gen_tcp
+
+    case module.controlling_process(state.socket, state.receiver_pid) do
       :ok ->
-        :ok = :inet.setopts(state.socket, active: :once)
+        if state.use_ssl? do
```

Comment:
Why not use `module.setops(...)` here?

Reply:
because `:ssl` module doesn't work with that